README.md

```diff
@@ -1,31 +1,36 @@
-Meta-kodi
-================================
+# Meta-kodi
 
-Introduction
--------------------------
+## Introduction
 
 The official OpenEmbedded/Yocto Project layer for Kodi
 
 The meta-kodi layer depends on:
 
-	URI: git://git.openembedded.org/openembedded-core
-	layers: meta
-	branch: daisy
+	URI: git://git.yoctoproject.org/poky.git
```

Comment:
Ehm, no. None of this poky shit in here. It depends on OE-core, not some toy fork of it.

Reply:
Point taken anyway, I was not aware that poky is fork of oe-core. I was introduced to world of bitbake recipes using poky any YP not long ago. This made me devil agent of poky I suppose ;-( I'll correct that.